lxd/storage/utils.go

```diff
@@ -456,7 +457,7 @@ func ImageUnpack(imageFile string, vol drivers.Volume, destBlockFile string, blo
 		// Use prlimit because qemu-img can consume considerable RAM & CPU time if fed a maliciously
```

Comment:
Please can you move the comment here into `RunQemuImgWithApparmor` so its kept with the actual call to `prlimit`.